cpp/src/arrow/util/compression-benchmark.cc

```diff
@@ -173,38 +172,22 @@ static void BM_StreamingDecompression(
 }
 
 template <Compression::type COMPRESSION>
-static void BM_StreamingDecompression(
+static void StreamingDecompression(
     benchmark::State& state) {                        // NOLINT non-const reference
   auto data = MakeCompressibleData(8 * 1024 * 1024);  // 8 MB
 
-  BM_StreamingDecompression(COMPRESSION, data, state);
+  StreamingDecompression(COMPRESSION, data, state);
 }
 
-BENCHMARK_TEMPLATE(BM_StreamingCompression, Compression::GZIP)
-    ->Unit(benchmark::kMillisecond)
-    ->Repetitions(1);
-BENCHMARK_TEMPLATE(BM_StreamingCompression, Compression::BROTLI)
-    ->Unit(benchmark::kMillisecond)
-    ->Repetitions(1);
-BENCHMARK_TEMPLATE(BM_StreamingCompression, Compression::ZSTD)
-    ->Unit(benchmark::kMillisecond)
-    ->Repetitions(1);
-BENCHMARK_TEMPLATE(BM_StreamingCompression, Compression::LZ4)
-    ->Unit(benchmark::kMillisecond)
-    ->Repetitions(1);
-
-BENCHMARK_TEMPLATE(BM_StreamingDecompression, Compression::GZIP)
-    ->Unit(benchmark::kMillisecond)
-    ->Repetitions(1);
-BENCHMARK_TEMPLATE(BM_StreamingDecompression, Compression::BROTLI)
-    ->Unit(benchmark::kMillisecond)
-    ->Repetitions(1);
-BENCHMARK_TEMPLATE(BM_StreamingDecompression, Compression::ZSTD)
-    ->Unit(benchmark::kMillisecond)
-    ->Repetitions(1);
-BENCHMARK_TEMPLATE(BM_StreamingDecompression, Compression::LZ4)
-    ->Unit(benchmark::kMillisecond)
-    ->Repetitions(1);
+BENCHMARK_TEMPLATE(StreamingCompression, Compression::GZIP);
```

Comment:
I think these are pretty much reference benchmarks, unless we find out that compression performance is affected by our abstraction layer...